scalameta/parsers/js/src/test/scala/scala/meta/parsers/JSFacadeSuite.scala

```diff
@@ -193,5 +193,54 @@ class JSFacadeSuite extends FunSuite {
     check(parsed, expected)
   }
 
+  test("default dialect is Scala 2.11") {
+    val code =
+      """|List(
+         |  1,
+         |  2,
+         |)""".stripMargin
+    val parsedDefaultDialect = JSFacade.parseStat(code)
+    val expected = d(
+      "error" -> "illegal start of simple expression"
```

Comment:
I just noticed the position is not included, do you think it would be useful to have it? It can be a separate PR or part of this one.